action.yml

```diff
@@ -24,5 +24,22 @@ inputs:
     default: main
 
 runs:
-  using: 'docker'
-  image: 'Dockerfile'
+  using: "composite"
+  steps:
+    - name: Set up pixi
+      uses: prefix-dev/setup-pixi@ba3bb36eb2066252b2363392b7739741bb777659  # v0.8.1
+      with:
+        cache: true
+        cache-write: ${{ github.event_name == 'push' && github.ref_name == 'main' }}
+        # Avoid post cleanup errors if action run multiple times
+        post-cleanup: false
```

Comment:
If the action is run multiple times in the same workflow (like we do in the CI to test) then at the end of the workflow each of the job steps gets cleaned up sequentially. [As](https://github.com/prefix-dev/setup-pixi/blob/6efead024a876e5a222673996735f4972a443a66/action.yml#L58-L61) 

```yaml
  post-cleanup:
    description: |
      If the action should clean up after itself. Defaults to `true`.
      If `true`, the pixi environment, the pixi binary and the rattler files in ~/.rattler and ~/.cache/rattler are removed.
```

this will try to delete the same directory path multiple times and will error when those paths don't exist after the first clean up.

I'd like to find a smarter way around this, as I'm not sure if leaving things around is ideal, but I also don't think it is _bad_.

Reply:
currently this is the only workaround. there are some nieche use cases where it actually makes sense to run setup-pixi twice. we might want to open an issue for this